scripts/download-models.sh

```diff
@@ -39,8 +39,13 @@ image_classification_init() {
     curl -sLO "${url}"
   done
 
-  # for custom model downloaded from tf hub
-  curl -sL "https://tfhub.dev/google/lite-model/aiy/vision/classifier/birds_V1/3?lite-format=tflite" -o "lite-model_aiy_vision_classifier_birds_V1_3.tflite"
+  # for custom model downloaded from kaggle
+  bird_model_name="lite-model_aiy_vision_classifier_birds_V1_3"
+  kaggle_tflite_filename="3.tflite"
+  curl -sL "https://www.kaggle.com/api/v1/models/google/aiy/tfLite/vision-classifier-birds-v1/3/download" -o "${bird_model_name}.tar.gz"
```

Comment:
The reason for using the Kaggle website instead of the TensorFlow Hub is that TensorFlow Hub has migrated to Kaggle Models, as indicated [here](https://www.kaggle.com/tfhub-dev-faqs). This change caused the previous test to fail, so I updated the source to download the model from Kaggle directly.